### PR TITLE
Update versions and bug fix for v0.8.3

### PR DESCRIPTION
**Pull request by allyhawkins**
Here I'm updating the release version to be v0.8.3. 

I also did a test run and found a small bug where we were trying to access hvg's in the ADT AnnData objects, which don't exist. So I fixed that by using the `--hvg_name "none"` option.

### CODE REVIEW

File: bin/reformat_anndata.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
     default="highly_variable_genes",
     help=(
         "Indicate the name used to store highly variable genes associated with the PCA in the exported AnnData object."
-        " Use the value 'none' if no highly variable genes were not used."
+        " Use the value 'none' if no highly variable genes were used."
```
Comment (sjspielman) [inline bin/reformat_anndata.py:43]:
```suggestion
        "Use the value 'none' if no highly variable genes were used."
```

File: bin/sce_to_anndata.R [hunk 2]
```diff
@@ -232,5 +232,6 @@ if (!is.null(opt$feature_name)) {
         ")
       )
     }
+    message("Exported alt")
```
Comment (sjspielman) [inline bin/sce_to_anndata.R:235]: just confirming if you meant to keep this or if it was part of testing? ok either way as long as it's what you meant!

**Review by sjspielman — APPROVED** [review]
🦏

File: bin/sce_to_anndata.R (hunk 2 above)
Comment (allyhawkins) [reply bin/sce_to_anndata.R:235]: Yes I did mean to leave it in. We have an `exported RNA` message but no message here and I think it was helpful to see where things fail.

**Pull request merged**